test/commands/solution/generate_help_file_test.rb

```diff
@@ -6,7 +6,35 @@ class Solution::GenerateHelpFileTest < ActiveSupport::TestCase
     solution = create :concept_solution, exercise: exercise
 
     contents = Solution::GenerateHelpFile.(solution)
-    expected = "# Help\n\n## Running the tests\n\nRun the tests using `ruby test`.\n\n## Submitting your solution\n\nTODO: (Required) define generic submit instructions\n\n## Need to get help?\n\nTODO: (Required) define generic help text\n\nStuck? Try the Ruby gitter channel." # rubocop:disable Layout/LineLength
+    expected = <<~EXPECTED.strip
+      # Help
+
+      ## Running the tests
+
+      Run the tests using `ruby test`.
+
+      ## Submitting your solution
+
+      You can submit your solution using the `exercism submit log_line_parser.rb` command.
```

Comment:
If the solution file path is in a nested directory, it will be using slashes (`/`) for the path separator. I've just checked on my Windows machine, and submitting file paths with forward slashes also works on Windows.

Reply:
Nice :)